cicd/backend/pipeline/deploy_logger_buildspec.yaml

```diff
@@ -0,0 +1,59 @@
+version: 0.2
+
+env:
+  shell: bash
+  variables:
+    JAVA_TOOL_OPTIONS: -Dhttps.protocols=TLSv1.2
+
+phases:
+  install:
+    commands:
+      - echo Installing dependencies...
+      - cat /etc/lsb-release
+      - apt update -y
+        && apt-get -q -y install openjdk-17-jdk
+        && pip3 install jinja2
+        && curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
+        && chmod 700 get_helm.sh
+        && ./get_helm.sh
+
+  build:
+    commands:
+      - echo Build started on $(date)
+      - aws eks --region ap-southeast-2 update-kubeconfig --name $EKS_CLUSTER_NAME
+      - kubectl config set-context --current --namespace=$EKS_NAMESPACE
+      - echo "Creating service account for logger..."
+      - |
+        LOGGER_ROLE_ARN=$(aws cloudformation describe-stacks --stack-name $BASE_STACK_NAME \
+          --query "Stacks[0].Outputs[?OutputKey=='LoggerRoleArn'].OutputValue" --output text)
+      - printf "$(kubectl config current-context)\n"
+      - |
+        eksctl create iamserviceaccount --name logger-service-account-$CLEAN_BRANCH \
+          --namespace $EKS_NAMESPACE --cluster $EKS_CLUSTER_NAME --attach-role-arn $LOGGER_ROLE_ARN \
+          --approve --override-existing-serviceaccounts \
+          --tags "product=logger,environment=development,branch=${CLEAN_BRANCH}"
+      - echo "Service account created."
+      - echo "Deploying logger via helm..."
+      - cd helm
+      - export BUILD_TAG_FROM_ARTIFACT=$(cat "$CODEBUILD_SRC_DIR_BuildTagArtifact/build_tag.txt")
```

Comment:
This works fine, saw that the BUILD_TAG is also an exported variable from cicd/backend/pipeline/build_logger_buildspec.yaml If it's gets added as an import in the pipeline to this action then the BUILD_TAG var can be accessed directly